packages/fuel-gauge/src/transaction-summary.test.ts

```diff
@@ -631,6 +631,10 @@ describe('TransactionSummary', () => {
     });
   });
 
+  it.todo(
```

Comment:
Was this forgotten?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3857/commits/2fc67ec0d3bcf3b32c4e3ecec83a75e4758ecdc4